src/features/CourtCaseList/NotePreviewButton.tsx

```diff
@@ -0,0 +1,79 @@
+import ConditionalRender from "../../components/ConditionalRender"
+import { Dispatch, SetStateAction } from "react"
+import { createUseStyles } from "react-jss"
+
+const useStyles = createUseStyles({
+  buttonContainer: {
+    color: "#1d70b8",
```

Comment:
`#1d70b8` is used a few times in the codebase. We could save it in `src/utils/colours.ts` as a variable & reuse.

Reply:
We realised we do not require this JSS anymore so have removed it. Styling is done by GDS classes.